### PR TITLE
chore: remove archived packages

### PR DESCRIPTION
**Pull request by jeffbelser-okta**
### Description

This PR removes archived docs package and the `@okta/odyssey` package from the monorepo source

### CODE REVIEW

**Review by magizh-okta — APPROVED** [review]

**Comment by jeffbelser-okta** [comment]
cc @edburyenegren-okta want to make sure I get your blessing on this before we merge? I think we're ready for this change now?

**Review by edburyenegren-okta — APPROVED** [review]
As long as the tests pass, I think we're good.

If there's anything we need to dig out for docs or server-side Odyssey in the future, we can always use the git time machine.

**Pull request merged**
